### PR TITLE
PS-7828: Add job to test PS InnoDB Cluster with EC2 instances

### PR DESCRIPTION
**Pull request by nlaperc**
This PR moves the existing [`test-ps-innodb-cluster` job from the old Jenkins](https://jenkins.percona.com/job/test-ps-innodb-cluster/) to the ps80 Jenkins instance.

The corresponding PR for the code changes is at Percona-QA/package-testing#88; lots of additional detail there.

You can see the job running at https://ps80.cd.percona.com/job/wip-test-ps-innodb-cluster/, against the `wip-test-ps-innodb-cluster` branches.

### CODE REVIEW

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+]) _
+
+void installDependencies() {
+    sh '''
+        export PATH=${PATH}:~/.local/bin
+        sudo yum install -y git python3-pip jq
+        sudo amazon-linux-extras install ansible2
+        python3 -m venv venv
+        source venv/bin/activate
+        python3 -m pip install --upgrade pip setuptools wheel
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.groovy:13]: why do you update python modules via pip and not by packages from the repos?

File: ps/jenkins/test-ps-innodb-cluster.groovy [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            defaultValue: '3558242',
+            description: 'Short git hash for release'
+        )
+        choice(
+            name: 'TEST_DIST',
+            choices: [
+                'ubuntu-focal',
+                'ubuntu-bionic',
+                'ubuntu-xenial',
+                'debian-10',
```
Comment (EvgeniyPatlan) [inline ps/jenkins/test-ps-innodb-cluster.groovy:110]: As we are currently working on adding debian 11 maybe it is high time to add into job?

File: ps/jenkins/test-ps-innodb-cluster.groovy (hunk 1 above)
Comment (nlaperc) [reply ps/jenkins/test-ps-innodb-cluster.groovy:13]: As you can see [right above this change](https://github.com/Percona-Lab/jenkins-pipelines/pull/1121/files/909ba9ab56efc0df1de3d119450028c4ba759006#diff-a98375b5fad48e3d1ffbd6fbe274841c5fdc16d1fe4d89a58cca4777c0990aa9R9), I am already installing the latest available version of pip (and setuptools, which is its dependency) from the repos.

However, as always, the repos are still a couple versions behind, and pip changed how it chooses which package version to install from the given requirements in October last year; [in the package installation invocation after this line](https://github.com/Percona-Lab/jenkins-pipelines/pull/1121/files/909ba9ab56efc0df1de3d119450028c4ba759006#diff-a98375b5fad48e3d1ffbd6fbe274841c5fdc16d1fe4d89a58cca4777c0990aa9R14), the upgraded version correctly picks `testinfra-3.4.0`, whereas the outdated version from the repositories picks `testinfra-6.0.0` despite it not being compatible with `molecule-2.22`.

Upgrading pip using pip is a well-established practice, and `pip` itself will display a prompt when using it, if it's out-of-date, suggesting you upgrade it in this manner. Python distributions support it. Also, [this is how pip is upgraded for other projects in this repository which install Molecule](https://github.com/Percona-Lab/jenkins-pipelines/blob/master/vars/installMolecule.groovy#L8).

File: ps/jenkins/test-ps-innodb-cluster.groovy (hunk 2 above)
Comment (nlaperc) [reply ps/jenkins/test-ps-innodb-cluster.groovy:110]: Sure thing, I'll work on that and let you know when it's ready.

**Review by Sudokamikaze — APPROVED** [review]
LGTM
But please add Debian 11 support, in case if you need an AMI ID or my assistant -- feel free to DM me on Slack

File: ps/jenkins/test-ps-innodb-cluster.groovy (hunk 2 above)
Comment (nlaperc) [reply ps/jenkins/test-ps-innodb-cluster.groovy:111]: The packages needed are present now and the job works as expected: https://ps80.cd.percona.com/job/wip-test-ps-innodb-cluster/98/

**Pull request merged**